common/os_calls.c

```diff
@@ -2437,6 +2437,13 @@ g_strncasecmp(const char *c1, const char *c2, int len)
 }
 
 /*****************************************************************************/
+const char* APP_CC
+g_strstr(const char* c1, const char* c2)
+{
+    return strstr(c1, c2);
```

Comment:
Yes, g_pos() already wraps strstr(). I don't see any need to handle Win32 specially here.